pages/CheckingPageTable.js

```diff
@@ -4,17 +4,19 @@ class CheckingPageTable extends BasePage {
     
     get newCheckingConfirmation () {return $('#new-account-conf-alert')}
 
-    get switchChecking () { return $('.switch-label')}
+    get switchChecking () { return $('//*[contains(text(),"Individual Checking")]//ancestor::div[@class="card-body"]//span[@class="switch-handle"]')}
 
-    get checkingBody () { return $('.even')}
+    get switchChecked () { return $('//*[contains(text(),"Individual Checking")]//ancestor::div[@class="card-body"]//input[@class="switch-input"]')}
 
     get checkingTransactionsTableNextButton () {return $('#transactionTable_next')}
 
     get checkingTransactionsTablePreviousButton () {return $('#transactionTable_previous')}
 
     get checkingTableTextInfo(){return $('#transactionTable_info')}
 
-    get checkingTableFilter(){return $('tbody')}
+    get getCellFromForRow(){return $(`#transactionTable>tbody>tr:nth-child(${row})>td:nth-child(2)`)}
```

Comment:
You're not passing this the row parameter that the body of this method is requesting to build the selector string. Also, this is a regular async method as well, not a getter method. Transform it accordingly.